pep-0605.rst

```diff
@@ -460,48 +548,47 @@ interests of operating environments that are able to consume new releases as
+-------------------------------------------------------------
 
+Using the "a" and "b" initials for the proposed rolling releases is a design
+constraint imposed by some of the pragmatic aspects of the way CPython version
+numbers are published.
 
-Why rolling beta releases rather than something like "brisk release cadence"?
------------------------------------------------------------------------------
+Specifically, alpha releases, beta releases, and release candidates are reported
+in some places using the strings "a", "b", and "c" respectively, while in others
+they're reported using the hex digits ``0xA``, ``0xB``, and ``0xC``. We want to
+preserve that, while also ensuring that any ``Python-Requires`` constraints
+are expressed against the beta releases rather than the alpha releases (since
+the latter may not enforce the ``abi3`` stability requirements if two alpha
+releases occur in succession).
 
-Using the "b" initial for the proposed rolling releases is a design constraint
-imposed by some of the pragmatic aspects of the way CPython version numbers are
-published (specifically, alpha releases, beta releases, and release candidates
-are reported in some places using the strings "a", "b", and "c" respectively,
-while in others they're reported using the hex digits ``0xA``, ``0xB``, and
-``0xC``, and we want to preserve that, while also ensuring that the rolling
-releases are all sorted after any alpha releases, and before the first release
-candidate).
-
-However, there isn't anything forcing us to say that the "b" stands for "beta".
+However, there isn't anything forcing us to say that the "a" stands for "alpha"
+or the "b" stands for "beta".
 
 That means that if we wanted to increase adoption amongst folks that were
-only being put off by the "beta" label, then it could make sense to rebrand the
-rolling beta releases as "brisk" releases, emphasising the rate of release over
-the fact that the release includes APIs and ABIs that aren't necessarily stable
-yet.
+only being put off by the "beta" label, then it may make sense to emphasise
+the "ABI breaking" and "binary compatible" names over the "alpha" and "beta"
+names, giving:
+
+* 3.9.0a1: ABI breaking pre-freeze release
+* 3.9.0b2: binary compatible pre-freeze release
+* 3.9.0rc1: release candidate
+* 3.9.0: final release
 
-In the near term, limiting adoption to folks that are comfortable with the
-"beta" label would be a good thing, as initial adopters are likely to
-encounter unexpected consequences at the level of the wider Python ecosystem,
-and would need to take an active part in getting those issues resolved.
+That said, limiting initial adoption to folks that are comfortable with the
+"beta" label may be a good thing, as early adopters of the rolling pre-freeze
+releases are likely to encounter unexpected consequences at the level of the
+wider Python ecosystem, and would need to take an active part in getting those
+issues resolved.
```

Comment:
Maybe we could reframe this as "our expectation is that it will take time to build trust in the new labels, and so slower adoption of the pre-freeze releases due to lingering fears around the names is a good thing" rather than it being treated as "OMG nobody will use it because it says 'beta'"?

Reply:
Exactly - the *first* time we do this, we want to limit the audience to folks that are willing to be active participants in making it work in the real world.

If that proves feasible, then the second and subsequent times are when it makes sense to start removing incidental barriers to adoption (while still making it clear it isn't going to be suitable for everyone)